howtos/reset-root-password.rst

```diff
@@ -1,81 +1,128 @@
+it might require to adjust the boot device sequence (in the BIOS, or, if
+available, during the boot process by pressing Function Keys like **<F12>**,
+depending on the motherboard manufacturer).
+
+On the splash screen, select the **Failsafe** option and hit <Enter>. This will
+boot the system into the command line. If, by accident, it boots into the first
+option with a graphical UI, either reset the machine and try again, or attempt to
+open a terminal window on the desktop and proceed from there with the command
+line-based process.
+
+.. image:: /images/howtos/reset-root-password/pw01_splash_screen.png
+   :scale: 50%
    :align: center
 
-In the **Rescue** screen, use the Tab key to highlight the **Continue** button
-and hit enter.
 
-.. image:: /images/howtos/reset-root-password/rescue_screen.png
+Step 2: Log in and mount relevant devices and partitions
+--------------------------------------------------------
+
+After the boot completes the login prompt should appear (the screenshot
+is from a virtual machine, hence the *vbox* login). Right above the network
+interfaces are listed, though they won't be needed for this activity.
+
+.. image:: /images/howtos/reset-root-password/pw01_rescue_shell.png
    :scale: 80%
    :align: center
 
-Hit Enter key again on the next two screens to be dropped into the Shell.
+To log into the system, just type :code:`root` for the user and hit <Enter>.
+Now that the login is complete, a device and a few partitions need to be 
+mounted for the password reset to work.
+
+First, identify the device that houses the Rockstor Operating System, using:
 
-.. image:: /images/howtos/reset-root-password/rescue_shell.png
+.. code-block: console
+   lsblk
+   
+to identify all devices connected to the machine, and then, identify
+the one that's the system device (likely the one that has the smallest size).
+Here is a sample output, it will look different on every user's machine:
+
+.. image:: /images/howtos/reset-root-password/pw01_lsblk_output.png
    :scale: 80%
    :align: center
 
-Step 3: Reset password
-----------------------
-
-Once you are in the rescue shell, change the filesystem root ::
+In the above example **/dev/sda4** is the system device (largest partition on
+*sda*).
 
-  sh-4.2# chroot /mnt/sysimage
+Next mount that device:
 
-Now change the password with the *passwd* command ::
+.. code-block: console
+   
+   vbox:~# mount -o rw /dev/sda4 /mnt
+   
+Then create a few bind mounts, change the filesystem root directory for the 
+running process and finally mount these, so that the system can use them
+for the next step:
 
-  bash-4.2# passwd
+.. code-block:: console
+   
+   vbox:~# mount -o bind /proc /mnt/proc
+   vbox:~# mount -o bind /sys /mnt/sys
+   vbox:~# mount -o bind /dev /mnt/dev
+   vbox:~# chroot /mnt
+   vbox:~# mount -a
 
-Remove autorelabel file ::
+Step 3: Reset password
+----------------------
 
-  bash-4.2# rm -f /.autorelabel
+Now change the password with the *passwd* command ::
```

Comment:
```suggestion
Now change the password with the *passwd* command:
```

@Hooverdan96 I think the " ::" formatting directive is clashing with what follows which is the intended console block. In your prior use here the ".. code-block:: console" worked just dandy. But when it follows the similar directive of " ::" it turns out not as intended:

![first-console-block-fail](https://github.com/user-attachments/assets/990d19c7-505d-4b2a-be0b-32bac24dfebe)